aiohttp/web_urldispatcher.py

```diff
@@ -755,6 +756,9 @@ def routes(self):
     def named_resources(self):
         return MappingProxyType(self._named_resources)
 
+    def set_defaults(self, *, allow_head):
```

Comment:
Humm, ok I'll remove it if you prefer. But the current setup is wrong according to the RFC and without the option to set the default you have to explicitly set `allow_head=True` on every route you add:

[RFC 7231](https://tools.ietf.org/html/rfc7231#page-25):

> The server SHOULD send the same
   header fields in response to a HEAD request as it would have sent if
   the request had been a GET, except that the payload header fields
   (Section 3.3) MAY be omitted

For me, the simplest, clearest and most correct option would be to have `allow_head=True` by default.